server/graphdb/internal/services/template.go

```
@@ -0,0 +1,206 @@
+			continue
+		}
+
+		renderedMarkdown, err := renderKindInfoMarkdown(
+			relationship.KindInfos[index].Content,
+			relationshipContext,
+		)
+		if err != nil {
+			relationship.KindInfos[index].RenderedMarkdown = renderedMarkdown
+			relationship.KindInfos[index].TemplateError = err.Error()
+			slog.WarnContext(ctx, "Failed to render relationship kind info markdown",
+				"info_key", relationship.KindInfos[index].InfoKey,
```

golangci-lint (2.12.2)

[major] key-value pairs should not be used

(sloglint)